docs/packages/Parser.mdx

```diff
@@ -128,6 +129,12 @@ Specifies the version of ECMAScript syntax you want to use. This is used by the
 
 This option allow you to tell parser to act as if `emitDecoratorMetadata: true` is set in `tsconfig.json`, but without [type-aware linting](../linting/Typed_Linting.mdx). In other words, you don't have to specify `parserOptions.project` in this case, making the linting process faster.
 
+### `experimentalDecorators`
+
+> Default `undefined`.
+
+This option allow you to tell parser to act as if `experimentalDecorators: true` is set in `tsconfig.json`, but without [type-aware linting](../linting/Typed_Linting.mdx). In other words, you don't have to specify `parserOptions.project` in this case, making the linting process faster.
```

Comment:
[Docs] This paragraph is another motivator for why I think it'd be good to spin out a blog post explaining the changes. On its own / without seeing the `consistent-type-imports` docs, this doesn't explain itself very well. But having a deeper resource explaining the differences separate from a lint rule would be very helpful.